CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,124 @@
+    [] I have made corresponding changes to the documentation.
+    [] My changes generate no new warnings.
+    [] Any dependent changes have been merged and published in downstream modules.
+    [] I ran all tests with success and extended them if necessary.
+    [] I have updated the CHANGELOG.md file in the root folder of the package after the [UPCOMING] title and before the latest version.
+    [] I have tested my code on the test network.
+    ```
+
+6. Wait for the pull request to be reviewer by the team.
+7. Make changes to the pull request if the maintainer recommends them.
+8. Celebrate your success after your pull request is merged! You will be able to claim your POAP for the hard work and join our Developer Inner Circle community.
+
+**Disclaimer**
+
+It's OK if your pull request is not perfect (no pull request is). The reviewer will be able to help you fix any problems and improve it.
+
+### Tips and Tricks
+
+Windows users may need to install the [windows-build-tools](https://www.npmjs.com/package/windows-build-tools) before installing this project's dependencies.
+
+### Documentation
+
+The documentation on how to run the front-end locally will be found within the `docs` folder of every repository.
+
+You can also access our [Developer Portal](https://devs.aragon.org) where you’ll learn about our architecture, how it works, and some tutorials which may be useful.
+
+### Style guide and development processes
+
+You can find our documentation [Style Guide here](https://www.notion.so/Documentation-Style-Guide-07c88cec18114b0aac88e8f0ba289976).
+
+For the frontends, we use [prettier](https://prettier.io/) and [eslint](https://eslint.org/) to automatically lint and format the project.
+
+For the contracts, we use [eth-lint](https://github.com/duaraghav8/Ethlint) to automatically lint the project.
+
+Handy `npm` scripts (usually `npm run lint`) are provided at all levels to help automate these checks.
+
+We generally avoid adding external dependencies if they can be ported over easily, due to numerous NPM-related security issues in the past (e.g. `[event-stream](https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident)`).
+
+### Git branch convention
+
+Due to the unconventional release process of smart contracts, this repo utilizes a slightly different flow of git.
+
+The main problem is that smart contracts should be heavily scrutinized before a release, making the process cumbersome and unlike the release process for "normal" dependencies or apps. [See here](https://forum.aragon.org/t/git-branch-convention-for-aragon-repos/298/3) for a more detailed explanation.
+
+Thus, we use the following convention:
+
+- Any change that can be release immediately, base on `[master](https://github.com/aragon/aragon-apps/tree/master)`
+- Any change that should be staged for longer review (e.g. contract functionality changes pending security review), base on `[next](https://github.com/aragon/aragon-apps/tree/next)`
```

Comment:
So far, we're working with internal branches and then merging to `develop`. We may change this after releasing, but I don't think this is accurate now.